### PR TITLE
Release_pydantic_relocation

### PR DESCRIPTION
**Pull request by canwaf**
Now using csvcubed-pydantic, same as v0.1.0-rc1.

### CODE REVIEW

File: csvcubed-devtools/pyproject.toml [hunk 1]
```diff
@@ -7,7 +7,7 @@ name = "csvcubed-devtools"
 packages = [
   {include = "csvcubeddevtools"},
 ]
-version = "0.0.1"
+version = "0.1.0-rc1"
```
Comment (robons) [inline csvcubed-devtools/pyproject.toml:10]: I wouldn't do this. The version number is to be updated by the auto-tag-version-thingy as described in the release proceedure.

Comment (robons) [reply csvcubed-devtools/pyproject.toml:10]: Same for each of the other version changes you've made too.

Comment (canwaf) [reply csvcubed-devtools/pyproject.toml:10]: I didn't make any changes, I created the branch from v0.1.0-rc1. The new thing will overwrite them anyways.

Comment (robons) [reply csvcubed-devtools/pyproject.toml:10]: Ok, but you're merging it into main so it's confusing who sets/where the version gets set.

**Review by robons — APPROVED** [review]

**Pull request merged**